doc/ledger3.texi

```diff
@@ -9677,9 +9677,10 @@ to the main body of the documentation.
 
 @menu
 * Cookbook::
+* Baseline Test Types::
 @end menu
 
-@node Cookbook,  , Miscellaneous Notes, Miscellaneous Notes
+@node Cookbook, Baseline Test Types, Miscellaneous Notes, Miscellaneous Notes
 @section Cookbook
```

Comment:
@thdox I changed my mind about this and moved the _Baseline Test Types_ out of the appendix into the _"Writing Tests"_ section, see 114d61e739ac43180526fc875d234b17aae471d0.